src/layouts/EditResourcePage.jsx

```diff
@@ -97,11 +97,13 @@ export default class EditResourcePage extends Component {
   render() {
     const { match } = this.props;
     const { siteName, fileName, resourceName } = match.params;
+    const { title, date, type } = prettifyResourceFileName(fileName);
     const { editorValue, canShowDeleteWarningModal } = this.state;
+    console.log(match)
```

Comment:
Is this log statement a typo?

Reply:
Whoops forgot to remove, resolved in https://github.com/isomerpages/isomercms-frontend/pull/167/commits/e127053b110dfdee0646768b3058e6e61e769924